R/pkg/R/functions.R

```diff
@@ -870,6 +870,19 @@ setMethod("cosh",
             column(jc)
           })
 
+#' @details
+#' \code{cot}: Returns the cotangent of the given value.
+#'
+#' @rdname column_math_functions
+#' @aliases cot cot,Column-method
+#' @note cot since 3.3.0
+setMethod("cot",
```

Comment:
I think you should also add it into ` R/pkg/R/generics.R`, and add a simple test at `R/pkg/tests/fulltests/test_sparkSQL.R`.

Also, let's create a separate JIRA for this one.

Reply:
Thanks, just added. 
So, if I add a new function, should I make a JIRA ticket and PR separately just for R implementation?